src/main/java/com/altenheim/kalender/JavaFXLauncher.java

```diff
@@ -12,11 +13,11 @@
 {
     @Override
     public void start(Stage primaryStage) throws Exception 
-    { 
+    {
```

Comment:
Es gab ein paar Aufrufe, bisher überwiegend zum Testen in der InjectorFactory, die dort nicht hingeören. Also initialies GUI Setup und jetzt noch hinzugekommen die initiale Deserialisierung der Settings und Anwendung derselben vor dem Rendern. Deshalb ist das jetzt erstmal hier drin, wird evtl aber nochmal ausgelagert wenn das was mit JFX zu tun hat zu viel wird.